static/js/click_handlers.js

```diff
@@ -176,7 +176,13 @@ export function initialize() {
     // if on normal non-mobile experience, a `click` event should run the message
     // selection function which will open the compose box  and select the message.
     if (!util.is_mobile()) {
-        $("#main_div").on("click", ".messagebox", select_message_function);
+        $("#main_div").on("click", ".messagebox", (e) => {
+            if (e.target.classList.value === "zulip-icon zulip-icon-ellipsis-v-solid") {
+                // Don't select message when user is trying to click the ellipsis icon.
+                return;
+            }
```

Comment:
I think for this, the right fix is to make `is_clickable_message_element`  return true for this element.  

(I guess this worked before because we had `stopPropagation` on this widget before)

Reply:
We can delete this now, since you actioned this comment in the first commit.

Reply:
Merged the first commit as da11141eae0fbc400a9176caad004000e44fa610, after extending the commit message to explain the background a bit more clearly.